packages/lpa-questionnaire-web-app/src/validators/health-safety.js

```diff
@@ -0,0 +1,23 @@
+const { body } = require('express-validator');
```

Comment:
There doesn't appear to be unit tests for this validator

Reply:
Thank you. Added.